jep/0000/README.adoc

```diff
@@ -0,0 +1,232 @@
+
+// Use the script `set-jep-status <jep-number> <status>` to update the status.
+| Status
+| Not Submitted :information_source:
+
+| Type
+| Process
+
+| Created
+| 2018-07-17
+
+| BDFL-Delegate
+| :bulb: Link to github user page :bulb:
+
+//
+//
+// Uncomment if there is an associated placeholder JIRA issue.
+//| JIRA
+//| :bulb: https://issues.jenkins-ci.org/browse/JENKINS-nnnnn[JENKINS-nnnnn] :bulb:
+//
+//
+// Uncomment if discussion will occur in forum other than jenkinsci-dev@ mailing list.
+//| Discussions-To
+//| :bulb: Link to where discussion and final status announcement will occur :bulb:
+//
+//
+// Uncomment if this JEP depends on one or more other JEPs.
+//| Requires
+//| :bulb: JEP-NUMBER, JEP-NUMBER... :bulb:
+//
+//
+// Uncomment and fill if this JEP is rendered obsolete by a later JEP
+//| Superseded-By
+//| :bulb: JEP-NUMBER :bulb:
+//
+//
+// Uncomment when this JEP status is set to Accepted, Rejected or Withdrawn.
+//| Resolution
+//| :bulb: Link to relevant post in the jenkinsci-dev@ mailing list archives :bulb:
+
+|===
+
+== Abstract
+
+This document documents the Google Summer of Code (GSoC) budgeting process.
+
+== Specification
+
+=== Budget sources
+
+The budget is formed from...
+
+* Payments by Google according to the
+link:https://developers.google.com/open-source/gsoc/help/org-payments[Payments to Google Summer of Code Organizations ]
+* Other donations to the Jenkins project,
+which explicitly target GSoC
+
+All payments are transferred to the Jenkins account
+provided by the link:https://www.spi-inc.org/[Software In Public Interest] non-profit organization,
+or to other official Jenkins treasury account.
+
+=== Using the budget
+
+The Google Summer of Code budget can be used ONLY to promote Google Summer of Code in the Jenkins project
+or to fund the ongoing Google Summer of Code
+or Google Code-In projects.
+It should not be used for other purposes,
+unless confirmed with GSoC organization admins.
+
+Possible usages:
+
+1. Travel grants for GSoC students in order to participate in Jenkins community or industry events
+** Examples: Jenkins Area Meetups, Jenkins World,
+open-source conferences like FOSDEM, etc.
+** Travel grants may cover: hotel, transportation costs, visa fees, travel insurance, meals
+** Transportation costs may be covered only for economy-class travel
+** Travel grants do NOT cover: conference fees, alcohol, souvenirs, etc.
+2. GSoC Mentor Summit travel grants for GSoC mentors and org admins
+** The budget is provided according to the rules defined in link:https://developers.google.com/open-source/gsoc/help/org-payments[GSoC Org Payments] (as of 2018 - 2200 USD)
+** Mentor travel grants may cover: transportation costs, visa fees, meals
+** Transportation costs may be covered only for economy-class travel
+** Travel grants do NOT cover travel expenses to any other events
+** Travel grants do NOT cover the hotel costs as long as it covered by the event organizer
```

Comment:
How about: Travel grants do NOT cover expenses that are already covered event organizers, or other types of sponsors.